plugins/services/src/js/constants/VolumeTypes.js

```diff
@@ -1,6 +1,25 @@
 var VolumeTypes = {
-  PERSISTENT: "Persistent",
-  EXTERNAL: "External"
+  DSS: {
+    name: "DC/OS Storage Service",
+    type: "DC/OS",
```

Comment:
Hm, having `type` a string with a '\' doesnt feel like the best idea… If someone decides to change it for UX reasons, we need to adapt it in our codebase…

maybe rename type to `label` (like the prop) and create a new ENUM type?

Reply:
I had the same ideal even the same name `label`.

The reason I decided going with type was that we already use `type` as label in the codebase.

For example: Look at the file `MarathonUtil.js` line `37`

I can change if you strongly believe it's better, I'm indifferent in that case.

Reply:
hm, actually that line has to be changed 🤔 we definitely do not want to have an object assigned there…  If this one has to land asap I'm okay with a follow-up ticket which takes care of this refactoring, but we have to change it at some point.

Actually it feels like we have to rethink the whole Constants File, because even using `VolumeTypes.PERSISTENT.type` looks hacky… I'd like us to have a better approach, maybe a simple rename from `VolumeTypes` to `VolumeDefinitions` would be enough for now - but thats something i would need to think about a bit longer /cc @dcos/frontend-foundation

Reply:
Sounds good to me.